docker/Dockerfile.aarch64_cross_compile

```diff
@@ -0,0 +1,58 @@
+FROM centos:7.6.1810
+
+# Install requirements
+RUN  set -x && \
+  yum -y install epel-release && \
+  yum -y install wget tar git make autoconf automake libtool openssl-devel ninja-build gcc-c++
+
+# Install Java and Golang
+RUN yum install -y java-1.8.0-openjdk-devel golang
+ENV JAVA_HOME="/usr/lib/jvm/java-1.8.0-openjdk/"
+
+# Install aarch64 gcc 4.9 toolchain
+RUN set -x && \
+  wget https://releases.linaro.org/components/toolchain/binaries/4.9-2016.02/aarch64-linux-gnu/gcc-linaro-4.9-2016.02-x86_64_aarch64-linux-gnu.tar.xz && \
+  tar xvf gcc-linaro-4.9-2016.02-x86_64_aarch64-linux-gnu.tar.xz
+ENV PATH="/gcc-linaro-4.9-2016.02-x86_64_aarch64-linux-gnu/bin:${PATH}"
+
+# Install Maven
+RUN set -x && \
+  wget http://repos.fedorapeople.org/repos/dchen/apache-maven/epel-apache-maven.repo -O /etc/yum.repos.d/epel-apache-maven.repo && \
+  yum install -y apache-maven
+
+# Install CMake
+RUN yum install -y cmake3 && ln -s /usr/bin/cmake3 /usr/bin/cmake
+
+# Cross compile Apache Apr for aarch64
+RUN set -x && \
+  wget https://www-us.apache.org/dist//apr/apr-1.6.5.tar.gz && \
+  tar xvf apr-1.6.5.tar.gz && \
+  pushd apr-1.6.5 && \
+  ./configure --prefix=/opt/apr-1.6.5 --host=aarch64-linux-gnu ac_cv_file__dev_zero=yes ac_cv_func_setpgrp_void=yes apr_cv_tcp_nodelay_with_cork=yes ac_cv_sizeof_struct_iovec=8 && \
```

Comment:
```
CFLAGS='-O3 -fno-omit-frame-pointer -fPIC' ./configure --disable-shared --prefix=/opt/apr-1.6.5 --host=aarch64-linux-gnu ac_cv_file__dev_zero=yes ac_cv_func_setpgrp_void=yes apr_cv_tcp_nodelay_with_cork=yes ac_cv_sizeof_struct_iovec=8 && \
```

the code works

Reply:
I can't reproduce the error you hit. Here is the file in the Docker image I built.
![image](https://user-images.githubusercontent.com/10891919/77398091-c8135200-6de1-11ea-86fc-b4084f38114f.png)
Not sure why the .so file doesn't exist in your docker image

Reply:
fixed